components/aws/sagemaker/unit_tests/README.md

```diff
@@ -0,0 +1,25 @@
+# Unit tests for AWS Sagemaker KFP Components 
+
+These unit tests test the user input parsing part of the code.
+
+## How to run these tests
```

Comment:
This should tell users to just use the Docker container. Perhaps instead of putting the header inside the Dockerfile, we could move those instructions out to here?

Reply:
wrote both methods